packages/relay/src/lib/utils/bloom/index.ts

```diff
@@ -0,0 +1,93 @@
+/*-
+ *
+ * Hedera JSON RPC Relay
+ *
+ * Copyright (C) 2022-2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import { keccak256 } from 'ethereum-cryptography/keccak.js';
+
+const BYTE_SIZE = 256;
+
+export class Bloom {
+  bitvector: Uint8Array;
+
+  /**
+   * Represents a Bloom filter.
+   */
+  constructor(bitvector?: Uint8Array) {
+    if (!bitvector) {
+      this.bitvector = new Uint8Array(BYTE_SIZE);
+    } else {
+      if (bitvector.length !== BYTE_SIZE) throw new Error('bitvectors must be 2048 bits long');
```

Comment:
Would be nice to have a refference link states why logsBloom must be 2048 bits (256 bytes).